itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/CompactionPoolOnTezTest.java

```diff
@@ -138,8 +138,8 @@ public void testShowCompactionsContainsPoolName() throws Exception {
     List results = new ArrayList();
     driver.getResults(results);
     Assert.assertEquals(3, results.size());
-    Assert.assertEquals("CompactionId\tDatabase\tTable\tPartition\tType\tState\tWorker host\tWorker\tEnqueue Time\tStart Time\tDuration(ms)" +
-       "\tHadoopJobId\tError message\tInitiator host\tInitiator\tPool name\tTxnId\tNext TxnId\tCommit Time\tHighest WriteID", results.get(0));
+    Assert.assertEquals("Id\tDatabase\tTable\tPartition\tType\tState\tWorker host\tWorker\tEnqueue Time\tStart Time\tDuration(ms)" +
```

Comment:
sorry, Kirti, probably i didn't elaborate well and was lacking some context. We shouldn't change the existing headers except the Id. Also, we should update 'Highest WriteID' to camel-case 'Highest WriteId'. 
For those columns where we can't apply default CC+value approach we should have a mapping logic (i.e. HadoopJobId -> CC_HADOOP_JOB_ID).